README.rst

```diff
@@ -0,0 +1,78 @@
+
+Introduction
+============
+
+.. image:: https://readthedocs.org/projects/adafruit-circuitpython-DS18X20/badge/?version=latest
+
+    :target: https://circuitpython.readthedocs.io/projects/ds18x20/en/latest/
+
+    :alt: Documentation Status
+
+.. image :: https://img.shields.io/discord/327254708534116352.svg
+    :target: https://discord.gg/nBQh6qu
+    :alt: Discord
+
+CircuitPython driver for Dallas 1-Wire temperature sensor.
+
+Dependencies
+=============
+This driver depends on:
+
+* `Adafruit CircuitPython <https://github.com/adafruit/circuitpython>`_
+* `Adafruit OneWire <https://github.com/adafruit/Adafruit_CircuitPython_OneWire>`_
+
+Please ensure all dependencies are available on the CircuitPython filesystem.
+This is easily achieved by downloading
+`the Adafruit library and driver bundle <https://github.com/adafruit/Adafruit_CircuitPython_Bundle>`_.
+
+Usage Example
+=============
+
+.. code-block:: python
+
+    >>> import board
+    >>> from adafruit_onewire.bus import OneWireBus
+    >>> from adafruit_ds18x20 import DS18X20
+    >>> ow_bus = OneWireBus(board.D2)
+    >>> ds18 = DS18X20(ow_bus, ow_bus.scan()[0])
+    >>> ds18.temperature
+    27.9375
+
+API Reference
+=============
+
+.. toctree::
+   :maxdepth: 2
+
+   api
+
+Contributing
+============
+
+Contributions are welcome! Please read our `Code of Conduct
+<https://github.com/adafruit/Adafruit_CircuitPython_DS18X20/blob/master/CODE_OF_CONDUCT.md>`_
```

Comment:
cookiecutter assumes the X should be capitalized but the github repo is lower case. Let me know which you want and lets make sure its consistent.

Reply:
Let's got with capital X.